src/main/resources/META-INF/rewrite/examples.yml

```diff
@@ -3377,6 +3358,25 @@ examples:
           }
       }
     language: java
+- description: ''
+  sources:
+  - before: |
+      import com.google.common.base.MoreObjects;
+
+      class A {
+          Object foo(Object obj) {
+              return MoreObjects.firstNonNull(obj, "default");
+          }
+      }
+    after: |
+      import java.util.Objects;
+
+      class A {
+          Object foo(Object obj) {
+              return Objects.requireNonNullElse(obj, "default");
+          }
+      }
+    language: java
```

Comment:
Quick FYI: local recipe runs and platform recipe runs disagree about the order, but are each internally consistent. No idea why, but best to limit your changes to new additions if you're kind enough to run these locally. (No need to update, just FYI)